### ISSUE TITLE
Can't run SVGViewDemo 

### ISSUE THREAD

**Issue by errnull** [GLMap/svgpb#2]
> Hi. I'm trying to install svgpb tool by using this instruction:
> 
> > SVGView
> > Set of tools to render simple SVG files on iOS using Core Graphics
> > Prerequisites
> > To download and build protobuf just run make.
> > Build
> > svg2pb is used in Build Rule of SVGViewDemo target to create protobuf version of SVG images and it should be build first.
> > Launch Archive action on svg2pb target to build and install svg2pb into /usr/local/bin. You may need to update Developer Team to build it.
> > Run SVGViewDemo application
> 
> I'm successfully download and install protobuf by running make command.
> After that I'm launch archive action on svg2pb target and it's archived successfully.
> Then I'm trying to run SVGViewDemo but it failed.

> clang: error: no such file or directory: '/Users/mine/Desktop/SVG/SVGPB/svg.pb.cc'
> clang: error: no input files
![屏幕快照 2019-11-02 上午2 16 48](https://user-images.githubusercontent.com/28481578/68046431-d8357280-fd16-11e9-9a6a-e7b396b3d974.png)

To fix it I'm run this cmd in terminal:
```
$ /usr/local/bin/protoc 
--proto_path=/Users/errnull/Desktop/SVG/SVGPB/SVGView/protobuf 
--cpp_out=/Users/errnull/Desktop/SVG/SVGPB 
/Users/errnull/Desktop/SVG/SVGPB/SVGView/protobuf/svg.proto
```
Then I'm trying to run SVGViewDemo but it still failed.
>  'google/protobuf/port_def.inc' file not found
![屏幕快照 2019-11-02 上午2 30 22](https://user-images.githubusercontent.com/28481578/68047201-be952a80-fd18-11e9-86e6-8b098b0305be.png)
Hope my explanation clearly, if not please ask for additional information.

**Comment by molind** [comment]
Hello, and sorry for delayed response.

I've added Makefiles required to build libraries. Just build a project from Xcode to trigger build. You should see libs/download and libs/build folders appear.

**Comment by molind** [comment]
Fixed in a6b7df39cc308a4a53a9fcd5cc5d7efeb6bb453a

**Comment by errnull** [comment]
> Fixed in [a6b7df3](https://github.com/GLMap/svgpb/commit/a6b7df39cc308a4a53a9fcd5cc5d7efeb6bb453a)

thank you.
